src/renderer/coremods/experiments/plaintextPatches.ts

```diff
@@ -1,13 +1,28 @@
+import { init } from "src/renderer/apis/settings";
 import type { PlaintextPatch } from "src/types";
+import { type GeneralSettings, defaultSettings } from "../settings/pages";
+
+// TODO: see if we can import this from General.tsx
+const generalSettings = await init<GeneralSettings, keyof typeof defaultSettings>(
+  "rp-settings",
+  defaultSettings,
```

Comment:
There is a bug with importing defaultSettings from `General.tsx` that's causing this to be undefined when it's run. This is causing the general page to not work properly since default settings currently won't be changed if you create a second settings instance.

 Let's move these to some other file and change General.tsx to point there as well. Hopefully that should fix the issue.

Reply:
"these"?

Reply:
Sorry, the `defaultSettings`